.circleci/config.yml

```diff
@@ -52,7 +52,7 @@ commands:
   unit_test:
     description: 'Running unit tests'
     steps:
-      - run: yarn cross-env NODE_ICU_DATA=./vendors/node_modules/full-icu run jest --projects jest.test.config.js --maxWorkers=3 --reporters jest-silent-reporter
+      - run: yarn run cross-env NODE_ICU_DATA=vendors/node_modules/full-icu jest --projects jest.test.config.js --maxWorkers=3 --reporters jest-silent-reporter
```

Comment:
Nit: the run scripts use `npm` CI uses `yarn`? Maybe settle on one CLI?

Reply:
which run script do you mean?

Reply:
There is currently an issue with installing `full-icu` with `yarn`. See [issue](https://github.com/unicode-org/full-icu-npm/issues/9).

Reply:
Ah sorry :)